### PR TITLE
Render close serials asynchronously 

### PR DESCRIPTION
**Pull request by jmromer**
Refactors close-serials search results to render asynchronously on page load

Screen Shot 2019-10-06 at 5 27 19 PM

demo

### CODE REVIEW

File: app/serializers/bike_v2_serializer.rb [hunk 1]
```diff
@@ -42,32 +41,43 @@ def title
...
-    end
+  end
+
+  def url
+    "#{ENV["BASE_URL"]}/bikes/#{object.id}"
   end
 
   def is_stock_img
-    object.public_images.present? ? false : object.stock_photo_url.present?
+    object.public_images.blank? && object.stock_photo_url.present?
```
Comment (sethherr) [inline app/serializers/bike_v2_serializer.rb]: 👍

File: app/serializers/bike_v2_serializer.rb [hunk 2]
```diff
@@ -42,32 +41,43 @@ def title
...
   end
 
   def stolen_location
-    object.current_stolen_record && object.current_stolen_record.address_short
+    stolen_record = object.current_stolen_record
```
Comment (sethherr) [inline app/serializers/bike_v2_serializer.rb]: Why this change? What is wrong with the `address_short` method? If you're just adding the country here, why not add it to the address_short method?

Comment (jmromer) [reply app/serializers/bike_v2_serializer.rb]: acfadad2

File: app/models/stolen_record.rb [hunk 3]
```diff
@@ -105,18 +105,20 @@ def address(skip_default_country: false, override_show_address: false)
     ].reject(&:blank?).join(", ")
   end
 
-  # Doesn't include street
-  def address_short
-    [city, state&.abbreviation, zipcode].reject(&:blank?).join(",")
```
Comment (jmromer) [inline app/models/stolen_record.rb]: The only call sites for `StolenRecord#address_short` are in the v2 stolen record and bike serializers. This commit refactors to use `address_location` instead, which better accounts for international locations.
